src/components/commons/token/PoolPairSymbol.tsx

```diff
@@ -59,9 +59,11 @@ export function PoolPairSymbol(props: PoolPairSymbolProps): JSX.Element {
   return (
     <div className="flex items-center" data-testid={props.testId}>
       <IconA
+        data-testid="AddressTokenCard.TokenSymbol"
         className={classnames("absolute z-10", props.symbolSizeClassName)}
       />
       <IconB
+        data-testid="AddressTokenCard.TokenSymbol"
```

Comment:
I just saw this, should we have it differently with IconA? So you can select IconA or IconB

Reply:
rollback it to initial state as it has been used, 
the data-testid being taken from `AddressBalances.tsx`